### PR TITLE
Answer: 39 

### PR DESCRIPTION
**Pull request by alcaidio**
## Checklist for challenge submission

- [x] Start your PR message with Answer:${challenge_number}

### CODE REVIEW

File: docs/src/content/docs/challenges/angular/39-injection-token.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ sidebar:
 
 ## Information
 
-In this small application, we start with a `VideoComponent` containing a **1-second** timer. The development team decided to use a global constant to store the timer value: `DEFAULT_TIMER`. However, a few weeks later, the product team wants to add a new screen for phone calls called `PhoneComponent`, and we want to reuse the `TimerComponent`. However, the product team wants a timer of **2 seconds**. How can we achieve this?
+In this small application, we start with a `VideoComponent` containing a **1-second** timer. The development team decided to use a global constant to store the timer value: `TIMER`. However, a few weeks later, the product team wants to add a new screen for phone calls called `PhoneComponent`, and we want to reuse the `TimerComponent`. However, the product team wants a timer of **2 seconds**. How can we achieve this?
```
Comment (tomalaforge) [inline docs/src/content/docs/challenges/angular/39-injection-token.md:16]: If you want to update the documentation, you need to do it in another PR

File: apps/angular/injection-token/src/app/timer.token.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { InjectionToken } from '@angular/core';
+
+export const TIMER = new InjectionToken<number>('timer');
```
Comment (tomalaforge) [inline apps/angular/injection-token/src/app/timer.token.ts:3]: you can create a default factory function, this way you don't have to provide the timer if you use the default value.

File: docs/src/content/docs/challenges/angular/39-injection-token.md (hunk 1 above)
Comment (alcaidio) [reply docs/src/content/docs/challenges/angular/39-injection-token.md]: Yes sorry, i revert it.
